app/services/AlfrescoStore.java

```
@@ -152,11 +152,6 @@ public AlfrescoStore(Config configuration, WSClient wsClient) {
                 onBehalfOfUser,
                 "uploadandrelease/" + documentId,
                 ImmutableMap.of("author", onBehalfOfUser)).
-                exceptionally(error -> {
```

Again, remove code that hides errors